setup.cfg

```diff
@@ -16,17 +17,18 @@ classifiers =
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
     Programming Language :: Python :: 3
     Programming Language :: Python :: Implementation :: CPython
     Programming Language :: Python :: Implementation :: PyPy
 platforms = any
 license = MIT
 
 [options]
-scripts = git-filter-repo
```

Comment:
PyPA guidelines [recommend against](https://packaging.python.org/guides/distributing-packages-using-setuptools/#scripts) the `scripts` keyword. It's also redundant to specify the same name in `scripts` and `console_scripts` at the same time and would cause conflicts.